### PR TITLE
Download json

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/modals/SettingsModal/CustomThemeForm.js [hunk 1]
```diff
@@ -615,6 +636,11 @@ export default function CustomThemeForm() {
           <Button size="sm" onClick={resetCurrentTheme}>
             Reset values
           </Button>
+          <Button
+            title="Save a copy of your customized theme, you may send it to us to include as a main theme. Importing a theme in the browser is not yet supported."
+            size="sm ml-2" onClick={downloadJsonFile} href={"#"} style={{ 'opacity': '40%'}}>
```
Comment (ogroppo) [inline src/modals/SettingsModal/CustomThemeForm.js:641]:
```suggestion
            size="sm ml-2" onClick={downloadJsonFile} style={{ 'opacity': '40%'}}>
```
not being a link href is not required

File: src/modals/SettingsModal/CustomThemeForm.js [hunk 2]
```diff
@@ -11,13 +14,31 @@ export default function CustomThemeForm() {
     resetCurrentTheme,
     setCustomThemeProp,
   } = useContext(AppContext);
+  // const history = useHistory();
+  // const [downloadUrl, setDownloadUrl] = useState("#");
```
Comment (ogroppo) [inline src/modals/SettingsModal/CustomThemeForm.js:18]: kill 🔪

File: src/modals/SettingsModal/CustomThemeForm.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
-import React, { useContext, useEffect } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { Form, Col, Row, InputGroup } from "react-bootstrap";
 import { AppContext } from "../../App";
 import useDebounce from "../../lib/useDebounce";
 import Button from "react-bootstrap/Button";
+import ls from "local-storage";
+import { STORED_CUSTOM_THEME_PREFIX_KEY } from "../../constants/storage";
+// import { useHistory } from "react-router-dom";
```
Comment (ogroppo) [inline src/modals/SettingsModal/CustomThemeForm.js:8]: kill 🔪

**Review by ogroppo — APPROVED** [review]

File: src/modals/SettingsModal/CustomThemeForm.js (hunk 3 above)
Comment (mshd) [reply src/modals/SettingsModal/CustomThemeForm.js:8]: only line 8 right

File: src/modals/SettingsModal/CustomThemeForm.js (hunk 2 above)
Comment (mshd) [reply src/modals/SettingsModal/CustomThemeForm.js:18]: Sure

**Pull request merged**